### PR TITLE
feat: improve empty column markers (styling, toggle, tooltips, tests)

### PR DESCRIPTION
**Pull request by konradmichalik**
## Summary

- Align empty column button styling with toolbar theme (dark background, white text)
- Match dashed outline to content element outline pattern (`repeating-linear-gradient`)
- Add translated tooltips: "Create new content in 'Column Name' column" (EN/DE)
- Expose `window.FRONTEND_EDIT_DISABLED` so toggle hides empty column buttons
- Extract `jsBool()` helper to reduce cognitive complexity in `addSettingsConfig()`
- Add ddev test fixtures with `<xfe:columnTarget>` markers (pages with empty columns)
- Add unit tests for `ColumnTargetViewHelper` (5 tests) and `EmptyColumnService` (5 tests)

## Test plan

- [x] `ddev install 13` — visit `/two-columns` and `/empty-columns`, verify buttons appear
- [x] Toggle frontend editing off via toolbar — empty column buttons should disappear after reload
- [x] Hover over empty column button — tooltip shows column name
- [x] Switch to dark mode — buttons match toolbar styling
- [x] Enable `prefers-reduced-motion` — no animations on dashed outline
- [x] `composer test` — 161 tests pass
- [x] `ddev cgl sca` — PHPStan level 8 clean

## Release Notes

* **New Features**
  * Empty column buttons now display configurable, translated labels and tooltips for creating content in specific columns.

* **Documentation**
  * Added visual guide demonstrating the frontend "Create content" button for empty columns.

* **Style**
  * Redesigned empty column markers with dotted/striped borders replacing outlines.
  * Updated empty column action buttons to align with toolbar styling.

* **Tests**
  * Added unit test coverage for empty column detection and column targeting functionality.

### CODE REVIEW

**Comment by konradmichalik** [comment]
@coderabbitai review

File: Tests/Acceptance/Fixtures/demo-content.sql [hunk 1]
```diff
@@ -73,3 +75,24 @@ REPLACE INTO `tt_content` (`uid`, `pid`, `tstamp`, `crdate`, `deleted`, `hidden`
...
+(27, 9, UNIX_TIMESTAMP(), UNIX_TIMESTAMP(), 0, 0, 256, 'text', 0, 'Main Content Only', 0,
+'<p>This page uses the default layout. Only the main content column (colPos 0) has content. The border (colPos 3), content-before (colPos 8), and content-after (colPos 9) columns are empty.</p>\n<p>You should see "Create new content" buttons in the empty column areas when frontend editing is enabled.</p>');
+
+-- Assign backend layout "2 Columns 50/50" to page 8
+UPDATE `pages` SET `backend_layout` = 'pagets__2_columns_50_50' WHERE `uid` = 8;
+
+-- sys_template: add sitepackage template override for page templates with <xfe:columnTarget> markers
+REPLACE INTO `sys_template` (`uid`, `pid`, `tstamp`, `crdate`, `deleted`, `hidden`, `sorting`, `title`, `root`, `clear`, `config`) VALUES
+(2, 1, UNIX_TIMESTAMP(), UNIX_TIMESTAMP(), 0, 0, 512, 'Sitepackage Override', 0, 0,
+'page.10.templateRootPaths.100 = EXT:sitepackage/Resources/Private/Templates/Page/\n');
```
[the remark this answers is not in the document]
Comment (konradmichalik) [reply Tests/Acceptance/Fixtures/demo-content.sql:98]: @coderabbitai This is a false positive. The `DELETE FROM sys_template WHERE pid = 1` is intentional — TYPO3 13+ uses **site sets** and the `bootstrap_package` extension to provide TypoScript. A `sys_template` with `root=1` is not required in this setup. The fixture only needs the non-root override to register `templateRootPaths.100` for the sitepackage page templates. Manually verified with `ddev install 13` — pages render correctly.

File: Tests/Unit/Service/Content/EmptyColumnServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+                0 => [200],       // uid 0 should be skipped
+                -5 => [100],      // negative uid should be skipped
+                'abc' => [100],   // non-numeric should be cast to 0 and skipped
+            ],
+        ];
+
+        $result = $service->getEmptyColumns(1, 0, '/return', $requestData);
+
+        $containerResults = array_filter($result, static fn (array $r): bool => isset($r['containerUid']));
+        self::assertEmpty($containerResults);
```
[the remark this answers is not in the document]
Comment (konradmichalik) [reply Tests/Unit/Service/Content/EmptyColumnServiceTest.php]: @coderabbitai Good catch. Added `expects(self::never())->method('buildUriFromRoute')` so the test verifies that invalid UIDs (0, -5, 'abc') are filtered before reaching the URL-building path — not just that the end result is empty. Fixed in 898f834.
